src/quacc/runners/prep.py

```diff
@@ -138,7 +138,7 @@ def calc_cleanup(
         symlink_path.unlink(missing_ok=True)
 
 
-def terminate(tmpdir: Path | str, exception: Exception) -> Exception:
+def terminate(tmpdir: Path, exception: Exception) -> None:
```

Comment:
It'd be nice to also support `Path | str` for `tmpdir`. Maybe the better option is to leave the original type hint for that and instead do `tmpdir = Path(tmpdir)` in the function itself?

Reply:
That sounds good, I'll do that.